docs/flows/flows.puml

```diff
@@ -130,7 +134,27 @@ alt#Gold #DarkSeaGreen trace
   bitdaoadmin <-- bitdaogovernance: TxHash
 end
 
+newpage
+
+autonumber "Upgrades:[00]"
+
+== Upgrades ==
+
+alice -> forum: create forum upgrade proposal
+alice <- guarantor: positive/negative feedback
```

Comment:
question: why is a guarantor needed? In this context what are they guaranteeing? 

Could feedback come from the community, or if roles are used the corresponding role?